api2/pkg/server/server.go

```diff
@@ -37,41 +38,44 @@ import (
 )
 
 type Server struct {
-	MongoClient             *mongo.Client
-	AttributeStore          *attributes.Store
-	AttributeHandler        *attributes.Handler
-	AttributeClient         *attributes.Client
-	IndividualStore         *Individuals.Store
-	IndividualHandler       *Individuals.Handler
-	IndividualClient        *Individuals.Client
-	RelationshipTypeStore   *relationshiptypes.Store
-	RelationshipTypeHandler *relationshiptypes.Handler
-	RelationshipTypeClient  *relationshiptypes.Client
-	RelationshipStore       *relationships.Store
-	RelationshipHandler     *relationships.Handler
-	RelationshipClient      *relationships.Client
-	PartyStore              *parties.Store
-	PartyHandler            *parties.Handler
-	PartyClient             *parties.Client
-	PartyTypeStore          *partytypes.Store
-	PartyTypeHandler        *partytypes.Handler
-	PartyTypeClient         *partytypes.Client
-	CaseTypeStore           *casetypes.Store
-	CaseTypeHandler         *casetypes.Handler
-	CaseTypeClient          *casetypes.Client
-	CaseStore               *cases.Store
-	CaseHandler             *cases.Handler
-	CaseClient              *cases.Client
-	WebAppHandler           *webapp.Handler
-	HttpServer              *http.Server
-	TeamStore               *teams.Store
-	TeamHandler             *teams.Handler
-	TeamClient              *teams.Client
-	MembershipHandler       *memberships.Handler
-	MembershipStore         *memberships.Store
-	membershipsClient       *memberships.Client
-	KeycloakClient          *keycloak.Client
-	SessionManager          sessionmanager.Store
+	MongoClient                *mongo.Client
```

Comment:
How come are there so many changes ?

Reply:
I'm not sure to be honest, this came about when I merged with main